actors/builtin/storage_power/storage_power_actor_state.go

```diff
@@ -193,16 +199,16 @@ func (st *StoragePowerActorState) deductClaimedPowerForSector(s adt.Store, miner
 	if err = st.setClaimedPower(s, minerAddr, big.Sub(currentPower, sectorPower)); err != nil {
 		return err
 	}
-	return st.updatePowerEntriesFromClaimed(s, minerAddr)
+	return nil
 }
 
-func (st *StoragePowerActorState) updatePowerEntriesFromClaimed(s adt.Store, minerAddr addr.Address) error {
+func (st *StoragePowerActorState) computeNominalPower(s adt.Store, minerAddr addr.Address) (abi.StoragePower, error) {
```

Comment:
Aligned with the comments above, change the address parameter to be claimed power (and any other callers must look it up first).